### ISSUE TITLE
Fully Automatic Blendshape Generation for Stylized Characters

### ISSUE THREAD

**Issue by nmfisher** [FuxiVirtualHuman/FuxiVirtualHuman.github.io#1]
Hi - the paper "Fully Automatic Blendshape Generation for Stylized Characters" mentions that the code/data is available at https://whitneypanye.github.io but I can't see anything there - will that be published in this repository?

**Comment by P2Oileen** [comment]
I failed to find anything now. Did you find something related to this work? @nmfisher

**Comment by nmfisher** [comment]
No, I never got a response and couldn't find any references elsewhere. Have you tried e-mailing Pan Ye directly?

**Comment by P2Oileen** [comment]
> No, I never got a response and couldn't find any references elsewhere. Have you tried e-mailing Pan Ye directly?

I tried but not getting any response from the first author and professor Ye 😭 Anyway thank you for your kindly response
